<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XYSmoothCurveDockGeneralTab</class>
 <widget class="QWidget" name="XYSmoothCurveDockGeneralTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>639</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="13" column="3" colspan="2">
    <widget class="QComboBox" name="cbMode"/>
   </item>
   <item row="12" column="3" colspan="2">
    <widget class="QSpinBox" name="sbOrder">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lOrder">
     <property name="text">
      <string>Polynom order</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="lMode">
     <property name="text">
      <string>Padding mode</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3" colspan="2">
    <widget class="QComboBox" name="cbWeight"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lData">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Data:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lXColumn">
     <property name="text">
      <string>x-data</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lYColumn">
     <property name="text">
      <string>y-data</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lType">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="2">
    <widget class="KComboBox" name="cbType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="19" column="0" colspan="5">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="18" column="3" colspan="2">
    <widget class="QTextEdit" name="teResult"/>
   </item>
   <item row="20" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>266</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="20" column="4">
    <widget class="QPushButton" name="pbRecalculate">
     <property name="text">
      <string>Recalculate</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="lResults">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Results:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lSmooth">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Smooth:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lComment">
     <property name="text">
      <string>Comment</string>
     </property>
    </widget>
   </item>
   <item row="22" column="0">
    <widget class="QCheckBox" name="chkVisible">
     <property name="text">
      <string>visible</string>
     </property>
    </widget>
   </item>
   <item row="21" column="3" colspan="2">
    <spacer name="verticalSpacerGeneral">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>24</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="KLineEdit" name="leComment"/>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="KLineEdit" name="leName"/>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>13</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3" colspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3" colspan="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lWeight">
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lPoints">
     <property name="text">
      <string>Points</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3" colspan="2">
    <widget class="QSpinBox" name="sbPoints">
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lPercentile">
     <property name="text">
      <string>Percentile</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3" colspan="2">
    <widget class="QDoubleSpinBox" name="sbPercentile">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="17" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="3" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lLeftValue">
       <property name="text">
        <string>Left value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sbLeftValue">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lRightValue">
       <property name="text">
        <string>Right value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sbRightValue">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
